lib/MD_Parola/src/MD_Parola.h

```diff
@@ -2438,8 +2438,8 @@ class MD_Parola : public Print {
 #if STATIC_ZONES
   MD_PZone _Z[MAX_ZONES]; ///< Fixed number of zones - static zone allocation
 #else // if STATIC_ZONES
-  MD_PZone *_Z;           ///< Zones buffers - dynamic zone allocation
+  MD_PZone *_Z = nullptr;           ///< Zones buffers - dynamic zone allocation
 #endif // if STATIC_ZONES
   uint8_t _numModules;    ///< Number of display modules [0..numModules-1]
-  uint8_t _numZones;      ///< Max number of zones in the display [0..numZones-1]
+  uint8_t _numZones = 0;      ///< Max number of zones in the display [0..numZones-1]
```

Comment:
Hmm if you actually use `numZones -1` as index, then `0` is maybe not the best default value :)

Reply:
Well, an `uint8_t` would probably induce another error when initialized with -1... 😃

Reply:
> Well, an `uint8_t` would probably induce another error when initialized with -1... 😃

Yep, that's exactly what I meant with indexing upto `numZones -1`.
Not sure if you use it anywhere, but given the comment, it was the first idea popping up in my head.

Reply:
The library is started by calling the `begin()` method, that actually checks and initializes `_numZones` correctly, so in fact it _is_ always initialized correctly, but that's not detected by the PVS-Studio scan. Maybe I'd have marked it as a false positive. 0 is an appropriate initial value here.